src/drivers/nuopc/cime_driver/esm.F90

```diff
@@ -543,6 +553,7 @@ subroutine SetModelServices(driver, rc)
        call shr_nuopc_methods_Clock_TimePrint(Eclock_o,subname//'EClock_base',rc)
        if (shr_nuopc_methods_ChkErr(rc,__LINE__,u_FILE_u)) return
     end if
+    call shr_nuopc_memcheck(subname, 0, mastertask)
```

Comment:
This the place where memory is output every step from mastertask (each driver root task)